canonical/juju-dashboard
# WD-7354 - fix: Fix Audit Logs filter

## Done

- Fix `Cannot destructure property 'value' of 'option' as it is undefined` when accessing `Audit Logs` filter.

## QA

- Without the changes, access Audit Logs on JIMM and try to open the filter. Error `Cannot destructure property 'value' of 'option' as it is undefined` should be displayed on screen.
- With the changes, the error should disapear and filter should work as expected.

## Details
- https://warthogs.atlassian.net/browse/WD-7354

## Screenshots

![Screenshot from 2023-11-08 14-57-26](https://github.com/canonical/juju-dashboard/assets/108150922/b3902f54-78b7-49ab-9685-90f4236d5bb7)
